eng/scripts/automation/Invoke-MockTest.ps1

```diff
@@ -0,0 +1,50 @@
+#Requires -Version 7.0
+param(
+    [string]$inputJsonFile,
+    [string]$outputJsonFile
+)
+
+. (Join-Path $PSScriptRoot .. MgmtTestLib.ps1)
+Write-output "inputfile:$inputJsonFile, outfile:$outputJsonFile"
+$inputstring = Get-Content $inputJsonFile | Out-String
+Write-Output "input parameter: $inputstring"
+$inputJson = Get-Content $inputJsonFile | Out-String | ConvertFrom-Json
+$packageFolder = $inputJson.packageFolder
+$packageFolder = $packageFolder -replace "\\", "/"
+
+$runLocalMockServer = $false
+if ([string]::IsNullOrEmpty($inputJson.mockServerHost)) {
+    $runLocalMockServer = $true
+}
```

Comment:
Mock server host will compose the mock request or response with specific version of swagger. If we use a remote mock server, how can handle the difference swagger version we need when doing mock test?